src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js

```diff
@@ -71,12 +68,12 @@ function Panel({
   });
 
   const toggleEditingOn = () => {
-    dismissNag(Nag.FIRST_BREAKPOINT_ADD);
+    dismissNag(Nag.FIRST_BREAKPOINT_EDIT);
```

Comment:
This was named incorrectly. Once you toggle editing on, you've finished `FIRST_BREAKPOINT_EDIT`, not `FIRST_BREAKPOINT_ADD`.